.circleci/config.yml

```diff
@@ -47,7 +47,8 @@ commands:
             JWT_2FA_EXPIRY_TIME="$JWT_2FA_EXPIRY_TIME"
             JWT_2FA_SECRET="$JWT_2FA_SECRET"
             JWT_ACCOUNT_ACTIVATION_SECRET="$JWT_ACCOUNT_ACTIVATION_SECRET"
-            JWT_EXPIRY_TIME="JWT_EXPIRY_TIME"
+            ACCOUNT_ACTIVATION_TIME="$ACCOUNT_ACTIVATION_TIME"
```

Comment:
consider updating `README.md` with a section about what format do we expect in each environment variable or a comment in `example.env`